pkg/capacity/manager_test.go

```diff
@@ -556,3 +539,70 @@ func TestGetOptimalTargetClusterWhenCounterIsNotInitialized(t *testing.T) {
 	require.EqualError(t, err, "unable to get the number of provisioned spaces: counter is not initialized")
 	assert.Equal(t, "", clusterName)
 }
+
+func TestGetOptimalTargetClusterWithSpaceProvisionerConfig(t *testing.T) {
+	t.Run("explicitly disabled", func(t *testing.T) {
+		// given
+		toolchainStatus := NewToolchainStatus(
+			WithMember("member1", WithNodeRoleUsage("worker", 68), WithNodeRoleUsage("master", 65)),
+			WithMember("member2", WithNodeRoleUsage("worker", 68), WithNodeRoleUsage("master", 65)))
+
```

Comment:
it's just a detail - you could move it up to the parent test and reuse it in both sub-tests
```suggestion
	toolchainStatus := NewToolchainStatus(
		WithMember("member1", WithNodeRoleUsage("worker", 68), WithNodeRoleUsage("master", 65)),
		WithMember("member2", WithNodeRoleUsage("worker", 68), WithNodeRoleUsage("master", 65)))

	t.Run("explicitly disabled", func(t *testing.T) {
		// given
```